docs/diginstroom/sip/1.0/sip_structure/5_structure_package.md

```diff
@@ -1720,6 +1720,9 @@ TODO: figure out the IDs
         <premis:eventDetailInformation>
             <premis:eventDetail />
         </premis:eventDetailInformation>
+        <premis:eventOutcomeInformation>
+            <premis:eventOutcome>1</premis:eventOutcome>
```

Comment:
Hier volgen we de https://id.loc.gov/vocabulary/preservation/eventOutcome.html vocabulary, dus de waarden zijn `success`, `failure` en `warning`. Het `valueURI` attribuut kan ook toegevoegd worden, bv. https://id.loc.gov/vocabulary/preservation/eventOutcome/suc.html

Reply:
oh, merci! die vocabulary heb ik gemist.